I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 18, in script
    with fade
IOError: Couldn't find file 'audio/song.mp3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 18, in script
    with fade
  File "C:\Users\edmun\OneDrive\Desktop\renpy-7.3.5-sdk\renpy\ast.py", line 1389, in execute
    renpy.exports.with_statement(trans, paired)
  File "C:\Users\edmun\OneDrive\Desktop\renpy-7.3.5-sdk\renpy\exports.py", line 1601, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "C:\Users\edmun\OneDrive\Desktop\renpy-7.3.5-sdk\renpy\display\core.py", line 2251, in do_with
    clear=clear)
  File "C:\Users\edmun\OneDrive\Desktop\renpy-7.3.5-sdk\renpy\display\core.py", line 2702, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "C:\Users\edmun\OneDrive\Desktop\renpy-7.3.5-sdk\renpy\display\core.py", line 3130, in interact_core
    renpy.audio.audio.interact()
  File "C:\Users\edmun\OneDrive\Desktop\renpy-7.3.5-sdk\renpy\audio\audio.py", line 1107, in interact
    periodic()
  File "C:\Users\edmun\OneDrive\Desktop\renpy-7.3.5-sdk\renpy\audio\audio.py", line 1060, in periodic
    six.reraise(exc[0], exc[1], exc[2])
  File "C:\Users\edmun\OneDrive\Desktop\renpy-7.3.5-sdk\renpy\audio\audio.py", line 1038, in periodic_thread_main
    periodic_pass()
  File "C:\Users\edmun\OneDrive\Desktop\renpy-7.3.5-sdk\renpy\audio\audio.py", line 979, in periodic_pass
    c.periodic()
  File "C:\Users\edmun\OneDrive\Desktop\renpy-7.3.5-sdk\renpy\audio\audio.py", line 457, in periodic
    topf = load(self.file_prefix + filename + self.file_suffix)
  File "C:\Users\edmun\OneDrive\Desktop\renpy-7.3.5-sdk\renpy\audio\audio.py", line 73, in load
    rv = renpy.loader.load(fn)
  File "C:\Users\edmun\OneDrive\Desktop\renpy-7.3.5-sdk\renpy\loader.py", line 576, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'audio/song.mp3'.

Windows-8-6.2.9200
Ren'Py 7.3.5.606
Unremarkable Everyday 1.0
Mon May 18 23:04:09 2020
